charts/cla/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,8 @@
+{{- if .Values.serviceAccount.create -}}
```

Comment:
Why would we want to not create the serviceAccount? I think we should remove this option.

Reply:
agree, done